.travis.yml

```diff
@@ -13,7 +13,6 @@ install:
   - npm install
   - bower install
   - pulp --version
-  - psc --version
```

Comment:
Rather than removing this line, could you please use `purs --version` here instead? The purpose of it is to make it clear what versions of everything is being used in case that information becomes useful for debugging.

Reply:
yep